docs/en/operation/Server-Logging.md

```diff
@@ -0,0 +1,159 @@
+that are utilizing the Alluxio API, we recommend looking at the
+[client logging documentation]({{site.baseurl}}{% link en/advanced/Client-Logging.md %})
+
+Alluxio's logging behavior can be fully configured through the `log4j.properties` file found in the
+`conf` folder.
+
+## Log Location
+
+### Default Location
+
+By default Alluxio processes log files can be found under `{ALLUXIO_HOME}/logs/`.
+
+### Configuring the Log Location 
+
+The location of the logs on is determined by the `alluxio.logs.dir` property. This is a JVM
+property which cannot be set in the `alluxio-site.properties` file. See the
+[configuration settings page]({{site.baseurl}}{% link en/advanced/Configuration-Settings.md %}#configuration-sources)
+for more information on how to set JVM properties for Alluxio.
+
+Each Alluxio process (master, worker, FUSE, proxy) will log to a separate file within the
+`alluxio.logs.dir` directory
+
+## Log Levels
+
+### Modifying Logging with `log4j.properties`
+
+You can modify the `log4j.properties` found under the `{ALLUXIO_HOME}/conf/log4j.properties` to modify logging levels.
+
+### Modifying Logging at Runtime
+
+Alluxio shell comes with a `logLevel` command that allows you to get or change the log level of a
+particular class on specific instances.
+
+The synax is `alluxio logLevel --logName=NAME [--target=<master|worker|host:port>] [--level=LEVEL]`,
+where the `logName` indicates the logger's name, and `target` lists the Alluxio masters or
+workers to set. If parameter `level` is provided the command changes the logger level, otherwise it
+gets and displays the current logger level.
+
+For example, this command sets the class `alluxio.heartbeat.HeartbeatContext`'s logger level to
+DEBUG on master as well as a worker at `192.168.100.100:30000`.
+
+```bash
+alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=master,192.168.100.100:30000 --level=DEBUG
+```
+
+And the following command gets all workers' log level on class `alluxio.heartbeat.HeartbeatContext`
+```bash
+alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=workers
+```
+
+## Remote Logging
+
+### Overview
+
+Alluxio supports sending logs to a remote log server over the network. This feature can be useful
+to system administrators who have to perform the task of log collection. With remote logging, the
+log files, e.g. master.log, worker.log, etc. on all Alluxio servers will be readily available on
+a designated and configurable directory on the log server.
+
+### Deploying The Log Server
+
+#### Start Log Server
+
+On the log server, execute the following command.
+
+```bash
+$ ./bin/alluxio-start.sh logserver
+```
+
+#### Stop the Log Server
+
+```bash
+$ ./bin/alluxio-stop.sh logserver
+```
+
+### Configuring Alluxio Processes to use the Log Server
+
+Refer to [Running Alluxio on a cluster]({{site.baseurl}}{% link en/deploy/Running-Alluxio-On-a-Cluster.md %})
```

Comment:
this seems out of place, what do you think?

Reply:
It was there before but I don't think it really fits in either. Whether or not alluxio is running on a cluster or locally doesn't really change remote logging, does it?

Reply:
Right, local vs cluster doesn't change anything. I think maybe it was included originally to make the doc more tutorial-like. I vote we remove it